docs/howtoguides/how_to_display_usns_that_affect_the_system.rst

```diff
@@ -0,0 +1,32 @@
+How to display USNs that affect the system
+******************************************
+
+To display which USNs affect the system, run the command:
+
+.. code-block:: bash
+
+    $ pro vulnerability list --usns
+
+
+When running the command you should see an output like this:
+
+.. code-block:: text
+
+    Ubuntu Security Notices (USN):
+    VULNERABILITY  FIX AVAILABLE FROM  AFFECTED INSTALLED PACKAGES
+    USN-5292-3     esm-infra           snapd
+    USN-5352-1     esm-infra           libtasn1-6
+    USN-5593-1     esm-infra           libzstd1
+    USN-5707-1     esm-infra           libtasn1-6
+    USN-5720-1     esm-infra           libzstd1
+
+    Vulnerabilities with applied fixes:
+        1 applied via Ubuntu Security
+
+    Vulnerabilities with fixes available:
+        5 fixable via Ubuntu Pro
+
+Note that this output is similar to what we describe in 
+:ref:`pro vulnerability list output <pro-vulnerability-list>`.
+The main differance is that we don't have a ubuntu priority for a USN and therefore
+this is not displayed in the output of the command.
```

Comment:
Here, instead of having this text, I'd just have the link to the other content and some text like "To find out more about the output of the ``pro vulnerability list`` command, refer to our explanation page" - and then I'd add an anchor specifically to the USN section you want to point people to

Reply:
Happy to leave this as-is for now. I'll revisit all these pages again once we've done the second pass/replacing old `fix` content etc, to make sure all the links are pointing to the best places.